### PR TITLE
US10

### PR DESCRIPTION
**Pull request by Merhene**
- Updated home wallpaper
- Fixed links from the menu section
- Personnalized titles in light theme

### CODE REVIEW

File: templates/home/index.html.twig [hunk 1]
```diff
@@ -10,11 +10,11 @@
 	</nav>
 	<header class="header-scroll">
 		<div id="home-wallpaper">
-			<img id="home-wallpaper" src="{{ asset('build/images/WP2.png') }}" alt="Stan's picture taking out a stencil">
+			<img id="home-wallpaper" src="{{ asset('build/images/tattoingStan.png') }}" alt="Stan's picture taking out a stencil">
 		</div>
 		<div class="header-content">
 			<img src="{{ asset('build/images/white_logo.png') }}" alt="Logo de Stan Dottori">
-			<h2>Click to join Stan dottori's world</h2>
+			{# <h2>Click to join Stan dottori's world</h2> #}
```
Comment (benrcd) [inline templates/home/index.html.twig]: dead code

File: assets/styles/app.scss [hunk 2]
```diff
@@ -53,6 +53,24 @@ body {
...
+.fa-solid.fa-angles-down {
+    animation: test 0.8s linear infinite alternate;
+}
+
+.fa-solid.fa-angles-down:hover {
+    animation: test linear forwards;
+    animation-range: entry;
+}
+
+@keyframes test {
```
Comment (benrcd) [inline assets/styles/app.scss]: Rename variable

**Review by benrcd — CHANGES_REQUESTED** [review]

File: templates/home/index.html.twig (hunk 1 above)
Comment (Merhene) [reply templates/home/index.html.twig]: ok !

File: assets/styles/app.scss (hunk 2 above)
Comment (Merhene) [reply assets/styles/app.scss]: ok !

**Review by benrcd — APPROVED** [review]

**Pull request merged**